Components/Input/input.html

```diff
@@ -0,0 +1,143 @@
+            <li><a class="link-hover" href="../lists/lists.html">Lists</a></li>
+            <li><a class="link-hover" href="../navigation/navigation.html">Navigation</a></li>
+            <li><a class="link-hover" href="../modal/modal.html">Modal</a></li>
+            <li><a class="link-hover" href="../rating/rating.html">Rating</a></li>
+            <li><a class="link-hover" href="../toast/toast.html">Toast</a></li>
+            <li><a class="link-hover" href="../grid/grid.html">Grid System</a></li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <div class="content">
+      <i id="sidebar-toggle" class="fas fa-bars hamburger"></i>
+      <h1 class="content-title">Input</h1>
+      <p class="sub-content">
+        Inputs are necessary for user interaction with the website. Inputs are used to take information from the user,
+        and then save the data to the server, accordingly.
+      </p>
+      <div class="divider"></div>
+      <h1 class="content-title">Label Input</h1>
+      <p class="sub-content">Label Input has a label present at the top of the input field. They can be customized to
+        show error error-message, and change color accordingly.
+        <div class="content-container">
+          <div class="input-group">
+            <label>Username</label>
+            <input class="input-w" type="text">
+          </div>
+          <div class="input-group err">
+            <label>Password</label>
+            <input class="input-w" type="text">
+            <p class="err-msg">Wrong Password. Try again.</p>
+          </div>
+        </div>
+      </p>
+      <iframe class="iframe-h-45"
+        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=lucario&wt=none&l=htmlmixed&width=339&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522input-group%2522%253E%250A%2520%2520%250A%2520%2520%253Clabel%253EUsername%253C%252Flabel%253E%250A%2520%2520%253Cinput%2520class%253D%2522input-w%2522%2520type%253D%2522text%2522%253E%250A%2520%2520%250A%253C%252Fdiv%253E%250A%250A%253Cdiv%2520class%253D%2522input-group%2520err%2522%253E%250A%2520%2520%250A%2520%2520%253Clabel%253EPassword%253C%252Flabel%253E%250A%2520%2520%253Cinput%2520class%253D%2522input-w%2522%2520type%253D%2522text%2522%253E%250A%2520%2520%253Cp%2520class%253D%2522err-msg%2522%253EError%2520Msg%253C%252Fp%253E%250A%2520%2520%250A%253C%252Fdiv%253E"
+        sandbox="allow-scripts allow-same-origin">
+      </iframe>
+      <div class="divider"></div>
+      <h1 class="content-title">Placeholder Input</h1>
+      <p class="sub-content">Placeholder Input has a text present inside of the input field. They can be customized to
+        show error.
+        <div class="content-container">
+          <div class="input-group">
+            <label>Email</label>
```

Comment:
label and input should be form

Reply:
thanks for pointing it out i will change it asap!